src/views/v2/commerce/checkout.php

```diff
@@ -40,21 +40,8 @@ class="tribe-tickets__commerce-checkout"
 		<?php foreach ( $sections as $section ) : ?>
 			<?php $this->template( 'checkout/cart', [ 'section' => $section ] ); ?>
 		<?php endforeach; ?>
+		<?php $this->template( 'gateway/gateways' ); ?>
 		<?php tribe( 'tickets.editor.template' )->template( 'v2/components/loader/loader' ); ?>
-		<?php
-		tribe( 'tickets.editor.template' )->template(
-			'components/notice',
-			[
-				'notice_classes'  => [
-						'tribe-tickets__notice--error',
-						'tribe-tickets__commerce-checkout-notice',
-				],
-				'content_classes' => [ 'tribe-tickets__commerce-checkout-notice-content' ],
-				'title'           => __( 'Checkout Error!', 'event-tickets' ),
-				'content'         => __( 'Something went wrong!', 'event-tickets' ),
-			]
-		);
-		?>
```

Comment:
@codingmusician @juanfra is this removed intentionally? The notice component is not getting loaded into the checkout page anymore.